### PR TITLE
Quarters and fiscal years

### PR DESCRIPTION
**Pull request by sakjur**
These are almost probably not in Elastic, but @grafana has introduced new time formats for quarters and fiscal quarters/years.

One significant change is that this introduces multiple character time units. It's possible that there are other (better?) solutions available for that, I'm submitting a naïve approach and you can let me know if you'd like it some other way, if you're never going to accept this functionality and we should maintain a fork, or if this is an ok approach.

Another thing I sneaked into this PR is an update for `CONTRIBUTORS.md` because `go get` is now deprecated for installing binaries.

### CODE REVIEW

**Comment by sakjur** [comment]
(cc @agnestoulet)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ This library relies on [goyacc](https://godoc.org/golang.org/x/tools/cmd/goyacc)
 
 To install, run:
 
-* `go get golang.org/x/tools/cmd/goyacc modernc.org/golex`
+* `go install golang.org/x/tools/cmd/goyacc@latest`
+* `go install modernc.org/golex@latest`
```
Comment (jszwedko) [inline CONTRIBUTING.md:12]: 👍

**Review by jszwedko — COMMENTED** [review]
Thanks @sakjur ! Apologies for the delay in review. I lost track of this one. The changes look good to me. I think just the module docs need to be updated (around here: https://github.com/vectordotdev/go-datemath/blob/52e568b686b7846ba7de762dd41198fb62d88ed4/datemath.go#L19-L28)

**Review by jszwedko — APPROVED** [review]
Thanks again @sakjur !

**Pull request merged**